### ISSUE TITLE
SlidingScreen should have a speed option

### ISSUE THREAD

**Issue by samreid** [phetsims/twixt#8]
It looks like the speed is currently hard-coded to 2/sec, if I'm understanding this line properly:

```js
this.ratio = Math.min( 1, this.ratio + 2 * dt );
```

Discovered in https://github.com/phetsims/area-model-common/issues/99

**Comment by jonathanolson** [comment]
So I hate to do this so quickly, but should be deprecate SlidingScreen in favor of TransitionNode (particularly once it is reviewed/etc.)?

Transition supports providing durations of the animations.

**Comment by samreid** [comment]
Yep, we should delete SlidingScreen and replace usages with TransitionNode.
